app/client/src/features/form/components/fragments/Job.js

```diff
@@ -88,7 +90,7 @@ function Job({ highlights, index, addHighlight, removeHighlight }: Props) {
             </RoundButton>
             <RoundButton
               inverted
-              disabled={i !== highlights.length - 1}
+              disabled={i !== highlights.length - 1 || !isRemoveBtnEnabled }
```

Comment:
I'm actually not sure if that `disabled` condition I had previously was actually doing anything. You can probably just change this to:

```js
disabled={highlights.length === 1}
```